content/en/monitors/guide/notification-message-best-practices.md

```diff
@@ -0,0 +1,168 @@
+
+In all of the above, you should **include links** to relevant pages scoped down to the alerting context to give responders clear next steps.
+
+Lastly, make sure your notification is routed to the right person/right recipients. There are several ways to receive a notification with Datadog:
+
+* You can use your email address in the notification message to receive the notification directly in your inbox.  
+* You can use [integrations](https://docs.datadoghq.com/monitors/notify/#integrations) handles to send your notifications to your desired integration (e.g. Slack)
+
+In the following sections you will learn how to use advanced features to further enhance your monitor messages. 
+
+### Variables
+
+Using variables is a great way to tweak your notifications to receive just the right information that you need.   
+You have 2 types of variables:
+
+* Conditional variables that use "*if-else"* logic to tweak the context of the message depending on some conditions, such as the state of the monitor.  
+* Template variables to enrich your monitor notifications with contextual information.
+
+Variables are especially important in a **Multi-Alert** monitor. When it triggers, you want to know which group triggered the monitor, and that's where variables are handy.
+
+Example:  
+Let's say you are monitoring the CPU usage of your containers, grouped by host.  
+{{< img src="/monitors/guide/notification_message_best_practices/query_parameters.png" alt="Example monitor query of container.cpu.usage metric averaged by host" style="width:100%;" >}}
+
+A great variable to use would be **{{host.name}}** to know the exact host that  triggered the alert.
+
+### Conditional variables
+
+These variables allow you to tailor the notification message based on your needs and use case by building some branch logic.
+
+For example, if you'd like to get notified only if a certain group triggers, you can use {{\#is\_exact\_match}} variable.  
+Example:  
+`{{#is_exact_match "role.name" "db"}}`  
+  `` This displays if the host triggering the alert contains `db` ``  
+  `in the role name. @db-team@company.com`  
+`{{/is_exact_match}}`
+
+That is also a way to notify different people/groups depending on the group that triggered the alert.  
+Example:  
+`{{#is_exact_match "role.name" "db"}}`  
+  `` This displays if the host triggering the alert contains `db` ``  
+  `in the role name. @db-team@company.com`  
+`{{/is_exact_match}}`  
+`{{#is_exact_match "role.name" "network"}}`  
+  `` This displays if the host triggering the alert contains `network` ``  
+  `in the role name. @network-team@company.com`  
+`{{/is_exact_match}}`
+
+You can receive a notification as well if the group that triggered the alert contains a specific string.  
+Example:  
+`{{#is_match "datacenter.name" "us"}}`  
+  ``This displays if the region triggering the alert contains `us` (e.g. us1, us3, …). @us.datacenter@company.com``  
+`{{/is_match}}`
```

Comment:
````suggestion
```
{{#is_match "datacenter.name" "us"}}
  # The content displays if the region triggering the alert contains `us` (such as us1 or us3)@us.datacenter@company.com``  
{{/is_match}}
```
````